_config.yml

```diff
@@ -106,6 +106,12 @@ redirects:
   /tr/yeni-baslamak: /tr/baslarken
   /en/bitcoin-for-press: /en/press
   /de/ueber: /de/faq
+  /de/bitcoin-paper: /de/bitcoin-abhandlung
+  /de/developer-examples: /de/enwickler-beispiele
```

Comment:
Hey Will, made a small mistake there. Should be "entwickler-beispiele" and not "enwickler-beispiele". Already corrected it on Transifex

Reply:
@Komodorpudel Thanks! Will update here as well.